src/components/Heading/Heading.mdx

```diff
@@ -145,6 +146,28 @@ to `level="one"`.
 
 <Canvas of={HeadingStories.SizeStylesDeprecated} />
 
+## Overline and Subtitle
+
+The `Heading` component offers two additional text elements as part of the full
+heading lockup: overline and subtitle. When the `overline` or `subtitle` props
+are passed, the `Heading` component will handle the styling for the
+corresponding text element.
+
+Use the `overline` prop to set the text for an eyebrow text element that sits
+above the heading text.
+
+<Canvas of={HeadingStories.Overline} />
```

Comment:
Per @EdwinGuzman:
It looks like these are going down starting from H1 but it's the size. Can you include text that says something like the following examples all use h1 but the size decreases?